infrastructure/cache/manager.py

```
@@ -2,199 +2,230 @@
+        )
+
     def _get_cache_key(self, messages: list, **kwargs) -> str:
         """生成缓存键"""
         # 将消息和参数转换为字符串
-        cache_data = {
-            "messages": messages,
-            "kwargs": kwargs
-        }
+        cache_data = {"messages": messages, "kwargs": kwargs}
+
+        # 如果需要包含模型名称，添加到缓存数据中
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)